docs/source/user-guide/introduction.md

```diff
@@ -120,6 +121,7 @@ Here are some active projects using DataFusion:
 - [Parseable] Log storage and observability platform
 - [Polygon.io](https://polygon.io/) Stock Market API
 - [qv] Quickly view your data
+- [R2 Query Engine] Cloudflare's distributed engine for querying data in Iceberg Catalogs
```

Comment:
A good link might be https://blog.cloudflare.com/r2-sql-deep-dive/

Reply:
```suggestion
- [R2 Query Engine](https://blog.cloudflare.com/r2-sql-deep-dive/) Cloudflare's distributed engine for querying data in Iceberg Catalogs
```

great catch -- thanks!